### DISCUSSION TITLE
Why can't `Get.theme` trigger view update, when the app brightness mode?

### DISCUSSION THREAD

**Discussion by ShookLyngs** [jonataslaw/getx#1385] [Q&A]
When using `Get.theme.xxx` as value, it has different behaviour to the `Theme.of(context).xxx`.

When I use `Theme.of` as value, then I trigger `Get.changeTheme()` manually, the page updated normally.
But when I use `Get.theme` as value, the page cannot update normally, when the app brightness mode changed.

Is this normal?

**Comment by jonataslaw** [comment]
Yes, this is the expected behavior, if you expect changes you should use Get context extensions, like context.theme.
Get.theme, Get.height, Get.widget, can only be used within functions, within widgets you must use context.theme, context.width, context.height, etc.
